ssl/quic/quic_port.c

```diff
@@ -748,6 +748,10 @@ static void port_send_retry(QUIC_PORT *port,
     if (ok == 0)
         return;
 
+    /*
+     * TODO (QUIC SERVER) need to retry this in the event it return EAGAIN
```

Comment:
I am sorry for being nitpicking but this needs to be `TODO(QUIC SERVER)` to make it easily searchable. The added space might make it being missed when searching TODOs to resolve.